### PR TITLE
Project 8: Details modal

### PR DESCRIPTION
**Pull request by Nelitaa**
My partner and I implement a popup window that includes the project details on both desktop and mobile with these interactions:

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- The popup disappears when the user clicks (or taps) the close (X) button.

Also, we refactor the project section:

- We use a JavaScript array to store all of the information for all projects.
- For each project we store the following pieces of data in a JavaScript object: name, description, featured image, technologies, link to live version, and link to the source.
- We update the main page so the projects section is created dynamically using the information stored in that JavaScript object.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,229 +1,246 @@
...
+          <ul class="nav-pages">
+            <li class="nav-pages-hello"><a href="#presentation">Hello</a></li>
+            <li class="nav-pages-portofilo">
+              <a href="#portfolio">Portfolio</a>
+            </li>
+            <li class="nav-pages-about"><a href="#about">About</a></li>
+            <li class="nav-pages-contact"><a href="#contact">Contact</a></li>
+          </ul>
+        </div>
+        <div class="navigation">
```
Comment (tufoinnkuo10) [inline index.html:47]: - [ ] Great job so far. Please kindly check your project again as I can't see any project been displayed, or rather I can't find the `see project` button in the first place. please verify and resubmit so the next reviewer can check the pop-up. great job

What you have for mobile
Screenshot (522)
figma
Screenshot (521)



what you have for desktop

Screenshot (524)
figma

Screenshot (520)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Team

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (Nelitaa) [reply index.html:47]: Sorry, I don't finish to push the new version. Now it's ready. Thanks for your understanding.

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @Nelitaa 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
- [ ] It's better if when the modal window is open it takes the whole page and the page becomes unscrollable instead of scrolling all the way up to see the modal window 👍 
- [ ] Kindly fix the bug where the user can open many modal windows at once 💯 

Animation

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (GzimAsani) [inline index.html:47]: - [ ] Currently on mobile I can see only half of the modal and I cannot scroll down in order to see all of it. Kindly make it smaller or enable scrolling, just as you did for the desktop version
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello    @Nelitaa       :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with making the required changes from the  previous reviewer, there is just one small thing left to do:+1:  
 
 ## Required Changes 

- [ ] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**